src/CodeGen_LLVM.cpp

```diff
@@ -2030,6 +1982,230 @@ void CodeGen_LLVM::scalarize(Expr e) {
     value = result;
 }
 
+void CodeGen_LLVM::codegen_predicated_vector_store(const Call *op) {
```

Comment:
Would it be cleaner to pass the args of the call here, instead of the call op itself? I think all uses of this have a Call node that they've already partially validated as legal predicated stores.